lib/Runtime/Language/JavascriptExceptionOperators.cpp

```diff
@@ -238,6 +256,14 @@ namespace Js
         catch (const Js::JavascriptException& err)
         {
             exception = err.GetAndClear();
+
+#if ENABLE_NATIVE_CODEGEN
+            if (scriptContext->GetThreadContext()->GetTryCatchFrameAddr() != nullptr)
+            {
+                Assert(exception->GetExceptionContext() && exception->GetExceptionContext()->ThrowingFunction());
+                WalkStackForCleaningUpInlineeInfo(scriptContext, exception->GetExceptionContext()->ThrowingFunction());
```

Comment:
>exception->GetExceptionContext()->ThrowingFunction( [](start = 65, length = 51)

Clean this up as this parameter is not really used and is the wrong thing to pass here anyway